final README updates for 200.5 (#528)
Esri/arcgis-maps-sdk-kotlin-toolkit: microapps/MapViewCalloutApp/README.md

# MapView Callout Micro-app

> This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.

Reviewer: ```suggestion
This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at a given point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.
```

> This micro-app demonstrates the use of `Callout` with a composable `MapView`. `Callout` is a composable function which renders an empty composable Box placed at some point or GeoElement on a Map. The content of the composable Box is customizable, while the container of the Box is a stylable rectangular shape with a leader line positioned at the point of the tap or GeoElement passed into the `Callout` composable functions.

Reviewer 2: ```suggestion
This micro-app showcases the use of `Callout` with a composable `MapView `. `Callout` is a composable function that renders an empty composable Box at a specified point or GeoElement on a Map. The content of the composable Box is customizable, and the container is a stylable rectangular shape with a leader line positioned at the tap location or GeoElement passed into the `Callout` composable function.
```

Author: @eri9000 I committed @shubham7109 's suggestion and this suggestion is not out of date.

## Usage

The application starts with a choice of two screens

> * A `MapView` with `Callout`s placed at the location of user taps on the screen.

Reviewer 2: ```suggestion
* A `MapView` that displays a `Callout` at the location where the user taps on the screen
```

> * A `MapView` with `Callout`s placed on GeoElements tapped by the user.

Reviewer 2: ```suggestion
* A `MapView` that displays a `Callout` placed on the GeoElement tapped by the user.
```

> * For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).

Reviewer: ```suggestion
* For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose#display-a-callout).
```

> * For more information on the composable `Callout` component and how it works, see its section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).

Reviewer 2: ```suggestion
* For more information on the composable `Callout` component and how it works, see the section in the composable GeoView [Readme](../../toolkit/geoview-compose/README#display-a-callout).
```
